package.json

```
@@ -36,7 +37,9 @@
     "build-storybook": "build-storybook",
     "prepare": "husky install",
     "prettier:check": "prettier . --check",
-    "prettier": "prettier . --write"
+    "prettier": "prettier . --write",
+    "analyze": "source-map-explorer 'build/static/js/*.js'",
```

`analyze` должен запускать билд и следом эксплорер

```suggestion
    "analyze": "analyze:build && source-map-explorer 'build/static/js/*.js'",
```